products/models.py

```diff
@@ -0,0 +1,32 @@
+from django.db import models
+
+class MainCategory(models.Model):
+    main_category = models.CharField(max_length=45)
+
+class SubCategory(models.Model):
+    category 	  = models.CharField(max_length = 45)
+    main_category = models.ForeignKey(MainCategory, on_delete=models.CASCADE)
+
+class Products(models.Model):
+    name          = models.CharField(max_length = 50)
+    price         = models.IntegerField(default = 0)
+    img_url       = models.URLField(max_length = 200)
```

Comment:
column명을 명확하게 수정해주세요~
URLField는 default max_length가 200으로 알고 있습니다. 그렇기 때문에 max_length=200을 사용할 목적이라면, 속성을 주지 않아도 될 것 같습니다~
```suggestion
    image_url       = models.URLField()
```